lingua_franca/lang/parse_common.py

```diff
@@ -385,3 +386,23 @@ def replace_right(source, target, replacement, replacements=None):
             extract = extract_handler(to_parse, short_scale, ordinals)
     numbers.reverse()
     return numbers
+
+
+class DurationResolution(Enum):
```

Comment:
I don't think this enum should exist (or should at least be reduced in scope/split up). As noted in my comment below, the changes in the PR conflate the concepts of durations and offsets. I think it would be better to use two separate functions rather than using this enum to change the behavior _and_ return type of a single function. Separating the functions should result in cleaner logic, and consistent return types per function.